Specs/DateTime/Spanish/DateTimeModel.json

```diff
@@ -18092,8 +18070,7 @@
     "Context": {
       "ReferenceDateTime": "2016-11-07T00:00:00"
     },
-    "Comment": "RANGECRIT. Null resolution.",
-    "NotSupported": "dotnet, javascript, python, java",
+    "NotSupported": "javascript, python, java",
     "Results": [
```

Comment:
I just noticed there is a range case marked "CRIT Range support". Can yuo check it and see if the problem is related to this PR or a different one?

Reply:
the case is still failing, the problem seems to be caused by the single datepoints not being extracted correctly. Should I fix it in this PR?

Reply:
It can be a separate fix after the higher priority issues are done. Thanks for checking. I had forgotten why it had been classified differently from the ones in this issue.